games.yaml

```diff
@@ -2041,6 +2211,35 @@
         - image: http://pentagram.sourceforge.net/images/screenshots/thumbs/japanese.png
           url: http://pentagram.sourceforge.net/images/screenshots/japanese.png
 
+- name: Uncharted Waters
+  clones:
+    - name: Uncharted waters 2 remake
+      repo: https://github.com/Jackyjjc/Uncharted-waters-2-remake
+      url: http://jackyjjc.com/
+      added: 2015-04-05
+      info: active development, Java, GPL2
+
+- name: [Uninvited, "Uninvited (video game)"]
+  clones:
+    - name: uninvited
+      added: 2015-04-04
+      repo: https://github.com/DavidGriffith/uninvited
+      info: sporadic development, Inform, Artistic License 2.0
+
+- name: [Utopia, "Utopia_(video_game)"]
+  clones:
+    - name: Utopia (video game)
+      added: 2015-04-05
+      url: http://apps.gamejs.org/newutopia/
+      repo: https://github.com/gamejs/newutopia
+      info: stalled development, JavaScript
+
+- name: [Warlords II, "Warlords_(game_series)#Warlords_II"]
+  clones: FreeLords
```

Comment:
You have to put "FreeLords" as a `name` property in the mapping, in other case YAML just can't parse the file.